docs/tips-and-tricks.md

```diff
@@ -266,59 +266,94 @@ http://www.idris.fr/media/data/formulaires/fgc.pdf, mostly the section "Ajout,
 modification ou suppression de machines".
 
 
-### Tensorboard dev
+### Tensorboard
 
-Tensorboard is a nice tool to monitor the progress of your trainings. 
-Currently there are 3 (known) ways to use it in Jean Zay: 
+Tensorboard is a nice tool to monitor the progress of your trainings.
+Currently there are 3 (known) ways to use it in Jean Zay:
 
-#### Use `idrjup` command
+#### Run sshfs + Tensorboard locally
 
-The recommended way to use Tensorboard on Jean Zay, is to use the `idrjup`
-command and launch it from a jupyter notebook environment.
-This method is described in the [official docs](http://www.idris.fr/jean-zay/pre-post/jean-zay-jupyter-notebook.html).
+The idea is to:
 
-#### Mount a sshfs partition and read the logs locally
+- use your local machine (i.e. the machine you are typing on) which is outside
+  of Jean-Zay
+- use `sshfs` so that your Tensorboard logs (that are on Jean-Zay) are visible
+  on your local machine
+- run Tensorboard locally on your logs mounted with `sshfs`
 
-An alternative to the official solution is to mount a folder from Jean
-Zay, where you store the logs generated by Tensorboard, and use
-the Tesorboard web server locally to visualize them. Let's suppose that 
-you store logs in `$SCRATH/tb_logs/`
+For example if your Tensorboard logs are in `/your/jean-zay/tensorboard/log/folder/`
 
-1. Mount the folder locally: 
+1. Mount the folder locally:
 ```
-sshfs -o reconnect,ServerAliveInterval=15,ServerAliveCountMax=3,follow_symlinks \
-  jeanzay:/gpfsscratch/rech/your/project/tb_logs/ $HOME/jeanzay/logs
+sshfs \
+  -o reconnect,ServerAliveInterval=15,ServerAliveCountMax=3,follow_symlinks \
+  jean-zay:/your/jean-zay/tensorboard/log/folder \
+  /your/local/tensorboard/log/folder
 ```
 
-2. Install tensorboard in your host machine:
+2. Install Tensorboard on your local machine for example with pip:
 ```
 pip install tensorboard
 ```
 
-3. Launch tensorboard locally, in the host machine. 
+3. Launch Tensorboard on your local machine
 ```
 tensorboard --logdir=$HOME/jeanzay/logs
```

Comment:
`/your/local/tensorboard/log/folder` instead of `$HOME/jeanzay/logs`

Reply:
right I'll fix that (after my next meeting)! Feel free to do it in the meantime 😉

Reply:
Done